src/seedu/addressbook/data/person/Postalcode.java

```diff
@@ -0,0 +1,11 @@
+package seedu.addressbook.data.person;
+
+public class Postalcode {
+	private String postalcode; 
```

Comment:
Variable names must be in mixed case starting with lower case.

e.g. `line`, `audioSystem`